.github/workflows/pr.yaml

```diff
@@ -0,0 +1,60 @@
+name: PR checks
+on: [push, pull_request]
```

Comment:
Do we need both? It seems to run twice now unnecessarily.

I think `pull_request` is enough for CI purposes on PRs?

Reply:
You are right, I think running PRs is enough.